.circleci/config.yml

```diff
@@ -148,34 +148,15 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: php:8.0
+      - image: php:8.1
 
     working_directory: ~/repo
 
     <<: *test_and_cover
 
-  build_php73:
-    docker:
-      - image: php:7.3
-
-    working_directory: ~/repo
-
-    environment:
-      - COMPOSER_FLAGS: --prefer-lowest
-
-    <<: *test
-
-  build_php74:
-    docker:
-      - image: php:7.4
-
-    working_directory: ~/repo
-
-    <<: *test
-
   code_fixer:
     docker:
-      - image: php:8.0
+      - image: php:8.1
```

Comment:
PHP 8.0 is supported for another two months. Should we instead keep it and add 8.1 and 8.2?

Reply:
This was a quick try to make the tests pass, but it failed too: https://app.circleci.com/pipelines/github/Lullabot/drupal-symfony-lock/194/workflows/9d4f7b62-c092-4631-947d-37a4e9e6bf05/jobs/1308
I'll spend some time tomorrow trying to make them green, and I can update this to include all PHP versions. 👍